src/main/java/de/nadirhelix/guestbook/ImageController.java

```diff
@@ -4,72 +4,91 @@
 
 import javax.annotation.Resource;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.MediaType;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import de.nadirhelix.guestbook.image.dto.PostData;
 import de.nadirhelix.guestbook.image.facade.ImageFacade;
 
 /**
- * This Controller handles all requests coming from the editor.
+ * This Controller handles all requests for certain images.
  * 
  * @author Phil
  */
 @RestController
 @RequestMapping("/images")
 public class ImageController {
+	
+	private static Logger log = LoggerFactory.getLogger(ImageController.class); 
 
 	@Resource
 	private ImageFacade imageFacade;
 	
 	/**
-	 * Handles Post creation out of given PostData.
+	 * Retrieves a post image for a given id.
 	 * 
-	 * @param post
-	 * 			the {@link PostData} from request
-	 * @return {@link ResponseEntity}
-	 * @throws IOException 
+	 * @param id
+	 * 			the id of the post
+	 * @return a {@link ResponseEntity} of byte[]
 	 */
 	@GetMapping("/posts/{id:.+}")
 	public ResponseEntity<byte[]> getPostImage(@PathVariable("id") String id) {
 		try {
 			return ResponseEntity.ok().contentType(MediaType.IMAGE_PNG).body(imageFacade.getPostImage(id));
 		} catch (IOException e) {
-			// TODO generate log here
-			e.printStackTrace();
+			log.debug("Could not find post with id " + id, e);
```

Comment:
log.debug("Could not find post with id {}", id, e)

Reply:
leider gibt es keine Methode, die mir message format und exception anbietet